package.json

```diff
@@ -17,40 +17,44 @@
   "author": "Skroutz",
   "dependencies": {
     "bower": "~1.8.0",
-    "coffee-script": "^1.9.1",
-    "grunt": "^0.4.4",
-    "grunt-cli": "^0.1.13",
-    "grunt-contrib-coffee": "^0.13.0",
-    "grunt-contrib-compress": "^0.13.0",
-    "grunt-contrib-concat": "^0.5.1",
-    "grunt-contrib-copy": "^0.8.0",
-    "grunt-contrib-environment": "^0.1.1",
-    "grunt-contrib-uglify": "^0.8.0",
-    "grunt-hash": "^0.5.0",
-    "grunt-replace": "^0.8.0",
-    "grunt-shell": "^1.1.2",
-    "load-grunt-tasks": "^3.1.0",
-    "requirejs": "^2.1.16"
+    "coffee-script": "~1.12.4",
+    "grunt": "~1.0.1",
+    "grunt-cli": "~1.2.0",
+    "grunt-contrib-coffee": "~1.0.0",
+    "grunt-contrib-compress": "~1.4.1",
+    "grunt-contrib-concat": "~1.0.1",
+    "grunt-contrib-copy": "~1.0.0",
+    "grunt-contrib-environment": "~0.1.1",
+    "grunt-contrib-uglify": "~2.2.0",
+    "grunt-hash": "~0.5.0",
```

Comment:
I think we should keep the caret range here (`~0.5.0`), since it is considered common for breaking changes to be introduced when version changes from `0.x.y` to `0.n.m`, and `x > n`.

See section on caret (^) usage [here](https://docs.npmjs.com/misc/semver).